android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Fri Mar 20 15:26:09 MDT 2020
+#Wed Jun 24 10:49:48 MDT 2020
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
 distributionPath=wrapper/dists
-zipStorePath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-all.zip
```

Comment:
This is more than a doc change. Is it going to break our use of the old `apply plugin:` syntax?

Reply:
True, I just wanted to make sure I was writing directions for the latest versions of everything.
No, everything was compatible, the old syntax is no longer preferred, but is still acceptable.